requirements.txt

```diff
@@ -1,11 +1,20 @@
-boto3==1.28.40
-botocore==1.31.40
+# boto3==1.28.40
```

Comment:
SLES15 SP5 comes with Python 3.6. This puts limitations onto versions for dependencies. Newest dependencies already dropped support for Python 3.6, we have to downgrade.